rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.create/20_synthetic_source.yml

```diff
@@ -197,7 +197,7 @@ empty object with unmapped fields:
 ---
 disabled root object:
   - requires:
-      cluster_features: ["mapper.track_ignored_source"]
+      cluster_features: ["mapper.ignored_source.dont_expand_dots"]
```

Comment:
I think we only introduce this label for bwc tests, right? In that case add `gte_v8.16.0` to the cluster features array here. After back porting this cluster feature should then be removed from this test.